readme.md

```diff
@@ -38,7 +48,9 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
+A SQL database has a very well-defined schema (requires migrations to update) with a strong "table with rows" structure. It is better for more complicated data relationships (especially many-to-many relationships).
 
+A NoSQL database uses the concept of documents and nested objects to structure the information. It is more flexible to changes in the schema.
```

Comment:
:tada: Excellent! I might say that complicated data relationships are easier in a SQL database but it is possible to model very complicated structures with a NoSQL database